doc/src/build/transactions.md

```diff
@@ -37,14 +37,24 @@ In addition to the common metadata above, a publish transaction includes Package
 
 Native transactions are optimized versions of common Sui operations. Each native transaction is semantically equivalent to a specific Move call but has a lower gas cost.
 
-### Transfer
+### Transfer Object
 
-This transaction type transfers coins from the sender to the specified recipients.
+This transaction type transfers objects from the sender to the specified recipients.
 
-In addition to the common metadata above, a transfer transaction includes the following fields:
-* Input: An object reference pointing to a mutable object owned by the sender. The object must be of type `sui::coin::Coin<T>` with arbitrary `T`--that is, any fungible token. The gas input object from above cannot also appear as an object input.
-* Recipients: The addresses that will receive payments from this transfer. This list must be non-empty.
-* Amounts: A list of unsigned integers encoding the amount that each recipient will receive. This list must be the same length as the recipients list. Each amount will be debited from the input object, wrapped in a freshly created coin object, and sent to the corresponding recipient address. The value of the input object must be greater than or equal to the sum of the amounts.
+In addition to the common metadata above, a transfer object transaction includes the following fields:
+
+* Input: An object reference pointing to a mutable object owned by the sender. The object must be of type that allows for public transfers--that is, any type with the `store` ability. The gas input object from above cannot also appear as the object input.
+* Recipients: The address that will receive payments from this transfer.
```

Comment:
Is this really plural, Recipients? If so, are multiple addresses supported?

If the latter is also correct, I would address by making this:
"Recipients: The address(es)..."

Reply:
Ah should be a single address I think.

I think this file in general was really out of date (and still is)

Reply:
> Ah should be a single address I think.
> 
> I think this file in general was really out of date (and still is)

Thanks! Do you know who would be the best person to review this admittedly old overview of the transaction process?

Reply:
@lxfind ?